### PR TITLE
pages created

### PR DESCRIPTION
**Pull request by RikaShah**
# What's this PR do? (related issue if exists)
login and signup pages created
# Any context you want to provide on the implementation?

# Screenshots? (necessary if an UI/UX change)

screenshot from 2019-01-25 23-50-03
screenshot from 2019-01-25 23-50-08

### CODE REVIEW

File: views/signup.ejs [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
+	<div class="main-w3layouts wrapper">
+		<h1><u><b> SignUp CodeManiacs</b></u></h1>
+		<div class="main-agileinfo">
+			<div class="agileits-top">
+				<form action="#" method="post">
+                    <input class="text" type="text" name="Name" placeholder="Full Name" required="">
+					<input class="text" type="text" name="Username" placeholder="Username" required="">
+					<input class="text email" type="email" name="email" placeholder="Email" required="">
+					<input class="text" type="password" name="password" placeholder="Password" required="">
+					<input class="text w3lpass" type="password" name="password" placeholder="Confirm Password" required="">
```
Comment (Vishal1541) [inline views/signup.ejs]: Make the attribute `name` to:
Line18: name = "name"
Line19: name = "username"
Line20: name = "email"
Line21: name = "password"
Line22: name = "password2"
all in lowercase

File: views/login.ejs [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+			<div class="agileits-top">
+				<form action="#" method="post">
+					<input class="text email" type="email" name="email" placeholder="Email" required="">
+					<input class="text" type="password" name="password" placeholder="Password" required="">
+					<div class="wthree-text">
+						<div class="clear"> </div>
+					</div>
+					<input type="submit" value="SIGNUP">
+				</form>
+				<p>Don't have an Account? <a href="./signup.ejs"> SignUp Now!</a></p>
```
Comment (Vishal1541) [inline views/login.ejs]: It should be `<a href="/user/signup">`

File: views/signup.ejs [hunk 3]
```diff
@@ -1,24 +1,44 @@
...
+					<div class="wthree-text">
+						<label class="anim">
+							<input type="checkbox" class="checkbox" required="">
+							<span>I Agree To The Terms & Conditions</span>
+						</label>
+						<div class="clear"> </div>
+					</div>
+					<input type="submit" value="SIGNUP">
+				</form>
+				<p>Already have an Account? <a href="./login.ejs"> Login Now!</a></p>
```
Comment (Vishal1541) [inline views/signup.ejs]: It should be `<a href="/user/login">`

File: views/login.ejs (hunk 2 above)
Comment (RikaShah) [reply views/login.ejs]: Sir, this means I have to create a new folder named user and place the files further in it??

Comment (Vishal1541) [reply views/login.ejs]: No. The route for accessing this page is already http://localhost:3000/user/login and http://localhost:3000/user/signup Just need to change the href

Comment (Vishal1541) [reply views/login.ejs]: Run the app and click on Login. You can see the URL

**Pull request closed without merging**